target/linux/pistachio/creator-platform-default-cascoda.config

```diff
@@ -6,7 +6,7 @@ CONFIG_LIBCURL_OPENSSL=y
 CONFIG_OPENSSL_WITH_EC=y
 CONFIG_VERSIONOPT=y
 CONFIG_VERSION_NICK="Ci40"
-CONFIG_VERSION_REPO="http://downloads.creatordev.io/pistachio/marduk/packages"
+CONFIG_VERSION_REPO="http://downloads.creatordev.io/openwrt/ci40-v1.0.1/pistachio/marduk/packages"
```

Comment:
If we do this, then we have to manually bump this up before every release, right?

Moreover, https://github.com/CreatorDev/openwrt/commit/95aae795dedfa8b832c019bc8037bf5af0b96f44 would no longer be needless?